src/features/admin/evaluation-administrations/[id]/edit/edit-evaluation-administration-form.tsx

```diff
@@ -241,21 +244,22 @@ export const EditEvaluationAdministrationForm = () => {
               <Button onClick={handleSubmit}>Save</Button>
             </div>
           </div>
-          <Dialog open={showDialog}>
-            <Dialog.Title>Cancel & Exit</Dialog.Title>
-            <Dialog.Description>
-              Are you sure you want to cancel and exit? <br />
-              If you cancel, your data won&apos;t be saved.
-            </Dialog.Description>
-            <Dialog.Actions>
-              <Button variant='primaryOutline' onClick={toggleDialog}>
-                No
-              </Button>
-              <LinkButton variant='primary' to='/admin/evaluation-administrations'>
-                Yes
-              </LinkButton>
-            </Dialog.Actions>
-          </Dialog>
+          <Suspense fallback={<div>Loading...</div>}>
+            <EvaluationAdminDialog
+              open={showDialog}
+              title='Cancel & Exit'
+              description={
+                <>
+                  Are you sure you want to cancel and exit? <br />
+                  If you cancel, your data won&apos;t be saved.
+                </>
+              }
+              showLinkButton={true}
+              showSubmitButton={false}
+              onClose={toggleDialog}
+              linkTo='/admin/evaluation-administrations'
```

Comment:
And remove this also, just create a function for example, `handleCancel`, then inside that function handle the `navigate('/admin/evaluation-administrations')`  and pass that function to the `onSubmit` props